book/src/developers/designs/application_passwords.md

```diff
@@ -161,55 +154,61 @@ needed.
 
 Since application passwords are related to applications, on delete of an
 application all entries that have a bound application password should be removed
-from user accounts.
+from user accounts. The associated group will be automatically deleted too.
 
 ### Access controls
 
-The IDM administrators group can manage the applications and user's application
-passwords.
+The "Application administrators" group will manage the applications, and
+applications will allow "managed by" so that they can have delegated
+administration.
 
-Users can manage their own application passwords.
+The "Application user passwords administrators" group will be able to list the
+users's application passwords and delete them but only the users will be able
+to, additionally to listing and deleting, self-create their own application
+passwords.
 
 ### LDAP
 
-The bind DN regular expression needs to adjusted to detect and determine the
-bind dn if it is related to an application or not. The application bind dn
-regular expression will capture the user name, the label and the application
-name.
+As application passwords will replace binding with UNIX passwords, the
+set-ldap-allow-unix-password-bind domain setting will be removed.
```

Comment:
No? That flag exists for a reason, it's there as a transition mechanism so that as users start to roll this out they can selectively disable unix binds from being allowed. If we just remove that feature and flag we will break existing user set ups. We need to always consider upgrades and migration paths.

Reply:
You are right. Fixed.